src/preview.ts

```diff
@@ -31,25 +31,19 @@ const renderToCanvas: RenderToCanvas<RRWebFramework> = async (context, element)
   const response = await fetch(`${url}/${id}`);
   const snapshot = await response.json();
 
-  const iframe = document.createElement('iframe');
-  iframe.setAttribute('style', iframeStyle);
-  element.appendChild(iframe);
+  const htmlNode = snapshot.childNodes[1];
 
   // @ts-expect-error rebuild is typed incorreclty, cache and mirror are optional
-  await rebuild(snapshot, { doc: iframe.contentDocument });
+  const html = (await rebuild(htmlNode, { doc: document })) as HTMLElement;
 
-  // Wait a moment, then set the dimensions of the iframe
-  setTimeout(() => updateDimensions(iframe), 100);
+  document.children[0].innerHTML = html.innerHTML;
 
-  // Also update the dimension every time the window changes size
-  window.addEventListener(
-    'resize',
-    debounce(() => updateDimensions(iframe), 100)
-  );
+  // insert a couple of elements to fool SB
+  document.body.innerHTML += '<div id="storybook-root"></div><div id="storybook-docs"></div>';
```

Comment:
Is there anything else we need here? Is it possible to load styles or other resources outside of the document that might not be included in the document body?

Reply:
@jwir3 this line is just inserting some divs for SB's sake (actually @tevanoff and I decided to replace them with `script` tags in the `head`), but the main work is done here: 

![image](https://github.com/chromaui/archive-storybook/assets/132554/578b1ca1-5078-46b5-bb67-b73152bc37ba)

We replace the entire contents of the `html` element, so the `head` and the `body`.